docs/source/using-discovery/bash.rst

```diff
@@ -0,0 +1,867 @@
+
+To use the man command, simply type man followed by the name of the command or utility for which you want to view the manual page. For example, to view the manual page for the ls command, you would type the following::
+  
+  man ls
+
+The manual page will be displayed in a pager program such as less, which allows you to scroll through the text and search for specific information.
+
+The man command is a valuable tool for learning about new commands and utilities, and it is an essential resource for understanding how to use the terminal effectively. Whether you are a beginner or an advanced user, the man command provides the information you need to get the most out of your tools and utilities.
+
+Basic Commands
+==============
+.. note:: 
+  See `_getting_access` for instructions on having a Discovery user account created.
+
+``ssh`` - Connect to a remote machine using Secure Shell (SSH)::
+
+   ssh <user-name>@login.discovery.neu.edu
+
+Reference: `ssh(1) manual page`_
+
+.. note::
+   The ``~`` character is shorthand for specifying the home directory of the current user, i.e., ``~`` is the same as ``$HOME``.
+
+``echo`` - Display a message or the value of a variable.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+  echo "Hello, world!"
+  Hello, world!
+
+.. note:: 
+  Variable names are accessible by appending its name to ``$`` (e.g., ${VARIABLE_NAME}, where ``{}`` are optional, but safer.)
+
+``pwd`` - Print the current working directory.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+  pwd
+  /path/to/directory1
+
+.. note::
+  ``pwd`` is also stored as an environment variable, i.e., ``${PWD}``. Running ``echo ${PWD}`` prints the same output as ``pwd``, but has the advantage of being accessed as part of a file pointer (e.g., ``ls ${PWD}/directory2`` to print all contents of ``directory2`` in the working directory.)
+
+``ls`` - List the contents of a directory.
+
+.. code-block:: shell
+  :emphasize-lines: 2
+
+  ls
+  file1.txt file2.txt directory1
+
+``cd`` - Change the current working directory.
+
+.. code-block:: shell
+  :emphasize-lines: 3
+
+  cd ~/directory1
+  pwd
+  <$HOME>/directory1
+
+``mkdir`` - Create a new directory.
+
+.. code-block:: shell
+  :emphasize-lines: 3
+
+   mkdir directory2
+   ls
+   file1.txt file2.txt directory1 directory2
+
+``rm`` - Remove a file or directory.
+
+.. code-block:: shell
+  :emphasize-lines: 3
+
+   rm file1.txt
+   ls
+   file2.txt directory1 directory2
+
+.. note:: 
+  To remove a directory, use ``rmdir`` if the folder is empty. Otherwise, recursively delete the directory and all of its contents via ``rm -r <FOLDER_PATH>``.
+
+``cp`` - Copy a file or directory.
+
+.. code-block:: shell
+  :emphasize-lines: 3
+
+   cp file2.txt file3.txt
+   ls
+   file2.txt file3.txt directory1 directory2
+
```

Comment:
You might want to add a comment that it is 'cp <existing file> <copy of existing file>' . It you follow through your example you can see what is being added or removed but if someone just jumps to the copy command they might not see what they are doing with this command.